javaio/CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ else()
    target_compile_definitions(DIPjavaio PUBLIC DIP__JAVAIO_IS_STATIC)
 endif()
 
-install(TARGETS DIPjavaio DESTINATION lib EXPORT DIPlibTargets)
+install(TARGETS DIPjavaio EXPORT DIPlibTargets DESTINATION ${DIP_DESTINATIONS})
```

Comment:
Shouldn't this be without `DESTINATION`? `${DIP_DESTINATIONS}` already includes the keyword.

Reply:
True! Fixed it.